src/draw.ts

```diff
@@ -0,0 +1,9 @@
+export const renderDrawMessage = () => {
```

Comment:
I find it really odd to have a dedicated file for only the DRAW case, but let's not worry about it for now.